homeassistant/components/proximity/strings.json

```diff
@@ -1,3 +1,19 @@
 {
-  "title": "Proximity"
+  "title": "Proximity",
+  "entity": {
+    "sensor": {
+      "dir_of_travel": {
+        "name": "Direction of travel",
+        "state": {
+          "arrived": "Arrived",
+          "away_from": "Away from",
+          "not_set": "Not set",
```

Comment:
answered this above in https://github.com/home-assistant/core/pull/101497#discussion_r1397413488

Reply:
My impression is this state value is for the new sensor not the legacy sensor?

Reply:
ahhh ... did not recognized that this comment is on the `strings.json` file 🙈  yeah you are absolutly right - i think `not_set` can be removed at all, because the value of the sensor will be set to `None` and HA should automatically translate it to unknown

Reply:
cc86a20